platform/preferred_languages.cpp

```diff
@@ -1,4 +1,7 @@
 #include "platform/preferred_languages.hpp"
+#include "platform/settings.hpp"
```

Comment:
Settings should be checked in framework.cpp

Reply:
done